doc/man3/OSSL_PARAM_int.pod

```diff
@@ -311,7 +288,7 @@ available parameters:
     size_t foo_l;
     char bar[1024];
     size_t bar_l;
-    const OSSL_PARAM request[] = {
+    OSSL_PARAM request[] = {
         OSSL_PARAM_UTF8_PTR("foo", foo, 0, foo_l),
         OSSL_PARAM_UTF8_STRING("bar", bar, sizeof(bar), bar_l),
```

Comment:
This example doesn't look right. A bit too much upper case, and the last argument should be gone, shouldn't it?

Reply:
Nice catch, sizes are gone and the macro names fixed.